src/test/java/com/ncc/aif/NistExamplesAndValidationTest.java

```diff
@@ -84,13 +87,15 @@ void invalid() {
 
                 // test relation edge argument must have a compound justification
                 final Resource relationEdge = markAsArgument(model, relation,
-                        SeedlingOntology.GeneralAffiliation_APORA_Affiliate, entity, system, 1.0, utils.getAssertionUri());
+                        LDCOntology.GeneralAffiliation_ArtifactPoliticalOrganizationReligiousAffiliation_ControlTerritory_Controller,
```

Comment:
Should probably use GeneralAffiliation.ArtifactPoliticalOrganizationReligiousAffiliation_EntityOrFiller to keep it as consistent as possible. Only including this comment once, but it applies several times

Reply:
Fixed